datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+import React, { ReactNode } from 'react';
+import { Image, Typography } from 'antd';
+import { ANTD_GRAY } from '../entity/shared/constants';
+
+// Home page
+export const GLOBAL_WELCOME_TO_DATAHUB_ID = 'global-welcome-to-datahub';
+export const HOME_PAGE_INGESTION_ID = 'home-page-ingestion';
+export const HOME_PAGE_DOMAINS_ID = 'home-page-domains';
+export const HOME_PAGE_PLATFORMS_ID = 'home-page-platforms';
+export const HOME_PAGE_MOST_POPULAR_ID = 'home-page-most-popular';
+export const HOME_PAGE_SEARCH_BAR_ID = 'home-page-search-bar';
+
+// Search page
+export const SEARCH_RESULTS_FILTERS_ID = 'search-results-filters';
+export const SEARCH_RESULTS_ADVANCED_SEARCH_ID = 'search-results-advanced-search';
+
+// Entity profile tabs
+export const ENTITY_PROFILE_ENTITIES_ID = 'entity-profile-entities';
+export const ENTITY_PROFILE_PROPERTIES_ID = 'entity-profile-properties';
+export const ENTITY_PROFILE_DOCUMENTATION_ID = 'entity-profile-documentation';
+export const ENTITY_PROFILE_LINEAGE_ID = 'entity-profile-lineage';
+export const ENTITY_PROFILE_SCHEMA_ID = 'entity-profile-schema';
+
+// Entity profile sidebar
+export const ENTITY_PROFILE_OWNERS_ID = 'entity-profile-owners';
+export const ENTITY_PROFILE_TAGS_ID = 'entity-profile-tags';
+export const ENTITY_PROFILE_GLOSSARY_TERMS_ID = 'entity-profile-glossary-terms';
+export const ENTITY_PROFILE_DOMAINS_ID = 'entity-profile-domains';
+
+// Managed ingestion
+export const ADMIN_CREATE_INGESTION_SOURCE_ID = 'admin-create-ingestion-source';
+export const ADMIN_REFRESH_INGESTION_SOURCES_ID = 'admin-refresh-ingestion-sources';
+
+// Business glossary
+export const BUSINESS_GLOSSARY_INTRO_ID = 'business-glossary-intro';
+export const BUSINESS_GLOSSARY_CREATE_TERM_ID = 'business-glossary-create-term';
+export const BUSINESS_GLOSSARY_CREATE_TERM_GROUP_ID = 'business-glossary-create-term-group';
+
+// Domains page
+export const DOMAINS_INTRO_ID = 'domains-intro';
+export const DOMAINS_CREATE_DOMAIN_ID = 'domains-create-domain';
+
+// Users page
+export const USERS_INTRO_ID = 'users-intro';
+export const USERS_SSO_ID = 'users-sso';
+export const USERS_INVITE_LINK_ID = 'users-invite-link';
+export const USERS_ASSIGN_ROLE_ID = 'users-assign-role';
+
+// Groups page
+export const GROUPS_INTRO_ID = 'groups-intro';
+export const GROUPS_CREATE_GROUP_ID = 'groups-create-group';
+
+// Roles page
+export const ROLES_INTRO_ID = 'roles-intro';
+
+// Policies page
+export const POLICIES_INTRO_ID = 'policies-intro';
+export const POLICIES_CREATE_POLICY_ID = 'policies-create-policy';
+
+export type OnboardingStep = {
+    id?: string;
+    title?: string | ReactNode;
+    content?: ReactNode;
+    selector?: string;
+    style?: any;
+};
+
+export const onboardingConfig: OnboardingStep[] = [
+    {
+        id: GLOBAL_WELCOME_TO_DATAHUB_ID,
+        content: (
+            <div>
```

Comment:
I'd recommend extracting this very important step HTML into a separate file so its easy to change!

Reply:
It has been moved!

Reply:
tytyt